### PR TITLE
Changes for rockv13 support

### PR DESCRIPTION
**Pull request by guymaness2**
##### What does the change add or fix?
This is a general update to Bulldozer to support Rock version 13.

* Update list of dlls in /Packages/Rock folder to include everything needed for Rock v13 elements to process.
* Added support for new BenevolenceType entity.
* Updated all projects and nuget packages to work with .NET 4.7.2

---------

##### What does the change add or fix in a succinct statement that will be read by clients?

* Update list of dlls in /Packages/Rock folder to include everything needed for Rock v13 elements to process.
* Added support for new BenevolenceType entity.
* Updated all projects and nuget packages to work with .NET 4.7.2

---------

##### Who reported, requested, or paid for the change?

Internal

---------

##### Does this update or add options to the block UI?

none

---------

##### What files does it affect?

* Bulldozer.BinaryFile/Bulldozer.BinaryFile.csproj
* Bulldozer.BinaryFile/app.config
* Bulldozer.BinaryFile/packages.config
* Bulldozer.CSV/Bulldozer.CSV.csproj
* Bulldozer.CSV/CSVComponent.cs
* Bulldozer.CSV/Maps/Benevolence.cs
* Bulldozer.CSV/SampleCSVs/26 - BenevolenceRequest.csv
* Bulldozer.CSV/app.config
* Bulldozer.CSV/packages.config
* Bulldozer.Example/App.config
* Bulldozer.Example/Bulldozer.Example.csproj
* Bulldozer.Example/packages.config
* Bulldozer.FellowshipOne/Bulldozer.F1.csproj
* Bulldozer.FellowshipOne/app.config
* Bulldozer.FellowshipOne/packages.config
* Bulldozer.sln.kfs
* Bulldozer/App.config
* Bulldozer/Bulldozer.csproj
* Bulldozer/Properties/Resources.Designer.cs
* Bulldozer/Properties/Settings.Designer.cs
* Bulldozer/Utility/AddMethods.cs
* Bulldozer/packages.config
* packages/Rock/AngleSharp.dll
* packages/Rock/DotLiquid.dll
* packages/Rock/Elasticsearch.Net.dll
* packages/Rock/Microsoft.SqlServer.SqlClrProvider.dll
* packages/Rock/Nest.dll
* packages/Rock/OrcaMDF.Core.dll
* packages/Rock/OrcaMDF.Framework.dll
* packages/Rock/Rock.Common.dll
* packages/Rock/Rock.Lava.Shared.dll
* packages/Rock/Rock.Lava.dll
* packages/Rock/Rock.Version.dll
* packages/Rock/Rock.ViewModel.dll
* packages/Rock/Rock.Wpf.dll
* packages/Rock/Rock.dll
* packages/Rock/Serilog.Formatting.Compact.dll
* packages/Rock/Serilog.Sinks.File.dll
* packages/Rock/Serilog.dll
* packages/Rock/StackExchange.Redis.StrongName.dll
* packages/Rock/Twilio.dll

---------

##### Is it a breaking change for other versions/clients?

none

### CODE REVIEW

File: Bulldozer.CSV/SampleCSVs/26 - BenevolenceRequest.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
\ No newline at end of file
+BenevolenceRequestText,BenevolenceRequestDate,BenevolenceRequestId,BenevolenceRequestFirstName,BenevolenceRequestLastName,BenevolenceRequestEmail,BenevolenceRequestCreatedById,BenevolenceRequestCreatedDate,BenevolenceRequestRequestedById,BenevolenceRequestCaseWorkerId,BenevolenceRequestCellPhone,BenevolenceRequestHomePhone,BenevolenceRequestWorkPhone,BenevolenceRequestGovernmentId,BenevolenceRequestProvidedNextSteps,BenevolenceRequestStatus,BenevolenceRequestResultSummary,BenevolenceRequestAddress,BenevolenceRequestAddress2,BenevolenceRequestCity,BenevolenceRequestState,BenevolenceRequestZip,BenevolenceRequestCountry,BenevolenceType
+Requesting help paying rent,00:00.0,2,Barney,Rubble,barneyr@bedrock.com,10477,46:30.2,10444,,,,,,,Approved,,,,,,,,
+Need money to by new XBox,00:00.0,3,Mickey,Mouse,mickey@mouse.com,,47:03.4,10479,,,,,,,Denied,,,,,,,,
+Need help with groceries this month,00:00.0,4,Minnie,Mouse,minnie@mouse.com,,48:56.3,10480,,,,,,,Needs Assignment,,,,,,,,
+Help with medical bills,00:00.0,5,Alex,Keaton,,10477,14:40.2,4,,(789) 456-1230,,,,,Needs Screening,,123 Sesame Street,,New York,NY,12345,US,Medical
```
Comment (guymaness2) [inline Bulldozer.CSV/SampleCSVs/26 - BenevolenceRequest.csv:5]:
```suggestion
Requesting help paying rent,2020-04-09 00:00:00.000,2,Barney,Rubble,barneyr@bedrock.com,10477,2020-04-09 15:46:30.210,10444,,,,,,,Approved,,,,,,,,
Need money to by new XBox,2020-10-08 00:00:00.000,3,Mickey,Mouse,mickey@mouse.com,,2020-10-08 10:47:03.420,10479,,,,,,,Denied,,,,,,,,
Need help with groceries this month,2020-10-08 00:00:00.000,4,Minnie,Mouse,minnie@mouse.com,,2020-10-08 10:48:56.350,10480,,,,,,,Needs Assignment,,,,,,,,
Help with medical bills,2020-12-16 00:00:00.000,5,Alex,Keaton,,10477,2020-12-16 12:14:40.247,4,,(789) 456-1230,,,,,Needs Screening,,123 Sesame Street,,New York,NY,12345,US,Medical
```

**Comment by nateh777** [comment]
Please check all nuget package versions compared to Rock v13 package versions.

**Comment by nateh777** [comment]
Please check all nuget package versions compared to Rock v13 package versions.

**Comment by nateh777** [comment]
Please check all nuget package versions compared to Rock v13 package versions.

**Comment by nateh777** [comment]
Please test BinaryFile import with v13 changes as well.

**Comment by guymaness2** [comment]
Verified all nuget package version matched versions defined in Rock v13 packages.config file where overlapped (no change necessary). 
Successfully tested binary import.
I think we are ready to merge.

**Review by nateh777 — APPROVED** [review]

**Pull request merged**